aptly-rest/examples/scanner.rs

```diff
@@ -0,0 +1,44 @@
+use std::path::PathBuf;
+
+use anyhow::Result;
+use aptly_rest::utils::scanner::{self, Scanner};
+use clap::Parser;
+use futures::TryStreamExt;
+
+async fn scan(path: PathBuf) -> Result<()> {
```

Comment:
As a followup PR, can you add some documentation here to explain what this does, what this is actually scanning?

It does not actually interact with Aptly in any way, right?